gnd/src/main/java/com/google/android/gnd/ui/offlinearea/OfflineAreaListAdapter.java

```diff
@@ -60,7 +47,7 @@ public AddressNotFoundException(String message) {
       this.binding = binding;
       this.areas = areas;
       this.navigator = navigator;
-      this.geocoder = new Geocoder(binding.getRoot().getContext());
+      this.geocoder = new GeocodingManager(binding.getRoot().getContext());
```

Comment:
This seems like logic that would normally be called from the ViewModel. Also, it would be preferable to inject the `GeocodingManager` (in the VM) to make it easier to swap out in tests. Would this be possible?

Reply:
77aaed7